### ISSUE TITLE
Indent current procedure produces garbage

### ISSUE THREAD

**Issue by sshikov** [rubberduck-vba/Rubberduck#1126]
Sample result: everything after 1st **End Sub** is duplicated from source procedure.

```
Public Sub testExport()
    Dim proj_name As String
    proj_name = "VbaDeveloper"

    Dim vbaProject As Object
    Set vbaProject = Application.VBE.VBProjects(proj_name)
    Build.exportVbaCode vbaProject
End Sub

   Dim vbaProject As Object
    Set vbaProject = Application.VBE.VBProjects(proj_name)
    Build.exportVbaCode vbaProject
End Sub
```

**Comment by sshikov** [comment]
Build from next branch.

**Comment by retailcoder** [comment]
Yeah, I made a "first pass" translating the VB6 code to C#, and then I needed the indenter settings to go any further; basically the code currently doesn't even know the number of indent spaces to use.

Shortly put, it's a known issue, there's another ticket about it somewhere.

I got carried away from the indenter code, and fixed a bunch of grammar and resolver bugs instead. But I'll get to it, promise!
